app/src/main/java/org/simple/clinic/recentpatient/RecentPatientScreenItemTypes.kt

```diff
@@ -82,7 +87,13 @@ data class DateHeader(val date: LocalDate) : RecentPatientScreenItemTypes<DateHe
 
   @SuppressLint("SetTextI18n")
```

Comment:
This lint suppression is not required any more.

Reply:
It's still required by the `text` assignment line.